src/MessageDispatch.KurrentDB/KurrentDbSubscriber.cs

```diff
@@ -245,7 +274,9 @@ public async void Start()
                             IsLive = false;
                             break;
                     }
-                }
+                }
+
+                SetupCallbackTimer();
```

Comment:
Not certain on the best place for this within this method, but I think doing it here won't ever actually create the timer (I'm not sure what would cause the code to get to this section in the while loop tbh). I guess ideally we want it to start the first time we start a subscription. Would probably be worth getting a wider opinion on this though, we do need to avoid it creating a new timer on every subscription drop/creation though